sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.net6.0.cs

```diff
@@ -20,6 +20,19 @@ public partial class DownloadTransferValidationOptions
         public bool AutoValidateChecksum { get { throw null; } set { } }
         public Azure.Storage.StorageChecksumAlgorithm ChecksumAlgorithm { get { throw null; } set { } }
     }
+    public partial class ExpectContinueOptions
+    {
+        public ExpectContinueOptions() { }
+        public System.TimeSpan Backoff { get { throw null; } set { } }
+        public long ContentLengthThreshold { get { throw null; } set { } }
```

Comment:
This should probably be nullable if we're not requiring it.  Is this only for Auto?  Is it a min or a max?  Can we make the name more specific?

Reply:
This is for both currently. And yes this can be nullable. I'm not sure how to be more descriptive for this name though without becoming word soup. `ContentLengthApplyExpectHeaderThreshold` starts to get pretty long while we do have a good docstring for the property. I think this comes back to having a snappy name for the options as a whole rather than using "expect continue." If that gets solved, I can probably think of a better name here.